src/utils/pair_programming.py

```diff
@@ -1,67 +1,65 @@
+import csv
 from dataclasses import dataclass
 import random
 
 
 @dataclass
 class Developer:
-	name: str
-	slack_id: str
-	projects: list
+    name: str
+    slack_id: str
+    projects: list
 
+def read_developers():
+    l = list()
+    with open('developers.csv') as csv_file:
```

Comment:
If you run main.py it raises an error

```python
Traceback (most recent call last):
  File ".../pivotal/src/main.py", line 4, in <module>
    from utils.pair_programming import get_pair_programming_message
  File ".../pivotal/src/utils/pair_programming.py", line 24, in <module>
    developers = read_developers()
  File ".../pivotal/src/utils/pair_programming.py", line 14, in read_developers
    with open('developers.csv') as csv_file:
FileNotFoundError: [Errno 2] No such file or directory: 'developers.csv'
```

Since it tries to load a local `developers.csv` file while it is in `utils/developers.csv`
Perhaps could it be placed in a `data` folder?

I tested, and it works
```suggestion
    with open('./data/developers.csv') as csv_file:
```

Reply:
it's right. Tnx for tip